scripts/make-rules/tools.mk

```diff
@@ -55,7 +55,7 @@ install.ossutil:
 install.upx:
 	@wget https://github.com/upx/upx/releases/download/v3.96/upx-3.96-amd64_linux.tar.xz
```

Comment:
Do you want to consider multi-architecture support? For example, arm64

Reply:
Thanks for reminding, I'll implement it soon.